src/layouts/Layout.astro

```diff
@@ -20,16 +20,22 @@ const { title } = Astro.props;
 	</body>
 </html>
 <style is:global>
-	:root {
-		--accent: 124, 58, 237;
-		--accent-gradient: linear-gradient(45deg, rgb(var(--accent)), #da62c4 30%, white 60%);
-	}
 	html {
 		font-family: system-ui, sans-serif;
-		background-color: #F6F6F6;
+		background-color: #212121;
+		color: #cfb39d;
```

Comment:
Take advantage of tailwindcss's color system

See: https://tailwindcss.com/docs/customizing-colors

Reply:
https://tailwindcss.com/docs/theme this link is better